source/_config.yml

```diff
@@ -0,0 +1,61 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: "地震“学”科研入门教程"
+author: seismo-learn
+copyright : "2020–2022"
+#logo: logo.png
```

Comment:
Where shall we use logo.png? I see that a new figure is added.

Reply:
By default, the logo will be shown like 
<img width="264" alt="image" src="https://user-images.githubusercontent.com/3974108/168477582-14b90fdc-9469-452b-b49a-7a64f0e9748b.png">

I'm not sure if we want to have the logo here. I can remove the "logo.png" for now.

Reply:
I don't think we want such a logo.

Reply:
Removed.